### ISSUE TITLE
Custom CRF is broken on MacOS

### ISSUE THREAD

**Issue by 2600box** [cdgriffith/FastFlix#52]
When trying to set a custom CRF, this is the problem I encounter. I cannot enter text into the custom text box (where it is 30 in the screenshot).

Works normally if I set to 18.

<img width="659" alt="Screen Shot 2020-08-27 at 18 24 27" src="https://user-images.githubusercontent.com/6595566/91431204-a2a7da00-e892-11ea-9e00-989c4ffd4d80.png">

```
Traceback (most recent call last):
  File "/Users/glacier/.local/pipx/venvs/fastflix/lib/python3.8/site-packages/fastflix/plugins/hevc/settings_panel.py", line 206, in <lambda>
    self.widgets.crf.currentIndexChanged.connect(lambda: self.mode_update())
  File "/Users/glacier/.local/pipx/venvs/fastflix/lib/python3.8/site-packages/fastflix/plugins/hevc/settings_panel.py", line 227, in mode_update
    self.main.build_commands()
  File "/Users/glacier/.local/pipx/venvs/fastflix/lib/python3.8/site-packages/fastflix/widgets/main.py", line 815, in build_commands
    settings = self.get_all_settings()
  File "/Users/glacier/.local/pipx/venvs/fastflix/lib/python3.8/site-packages/fastflix/widgets/main.py", line 808, in get_all_settings
    settings.update(**self.video_options.get_settings())
  File "/Users/glacier/.local/pipx/venvs/fastflix/lib/python3.8/site-packages/fastflix/widgets/video_options.py", line 51, in get_settings
    settings.update(self.current_settings.get_settings())
  File "/Users/glacier/.local/pipx/venvs/fastflix/lib/python3.8/site-packages/fastflix/plugins/hevc/settings_panel.py", line 273, in get_settings
    settings.crf = int(crf.split(" ", 1)[0]) if crf != "Custom" else int(self.widgets.custom_crf.text())
ValueError: invalid literal for int() with base 10: '16Custom'
```

**Comment by cdgriffith** [comment]
Oh hey, I actually just fixed this in 2.4.1 and didn't notice a bug report was open for it. I'm leaving it open for the moment just to remember to add attribution with next release.

**Comment by cdgriffith** [comment]
Attributed in 2.5.0
